index.html

```diff
@@ -3,11 +3,55 @@
     <head>
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
-
-        <title>Hello Microverse</title>
-        <link rel="stylesheet" href="../Hello-Microverse/style.css">
+        <title>Setup and Mobile Version Skeleton</title>
+        <link rel="stylesheet" href="../setup-and-mobile-version-skeleton/style.css">
     </head>
-    <body>
-        <h1>Hello Microverse</h1>
+    <body >
+        <header id="header-container">
+            <div class="logo-container">My logo</div>
+
+            <nav>
+                
+                    <img class="Mobile-menu" src="../setup-and-mobile-version-skeleton/Union.png" alt="">
+                
+               <!-- <li>
+                    <a>Home</a>
+                    <a>Portfolio</a>
+                    <a>Contact</a>
+                </li>-->
+            </nav>
+
+        </header>
+        <div id="Header-background">
+
+            <section class="headline">
+
+              <div class="intro-container">
+                    <div class="headline-intro">
+                        <h1>I’m Massaillon
+                            Glad to see you!</h1>
+                    </div>
+                    <div class="intro-desc-sumary">I’m a software developer! I can help you build a product , 
+                        feature or website Look through some of my work and experience! 
+                        If you like what you see and have a project you need coded,
+                         don’t hestiate to contact me.  
+                    </div>
+                
+
+                <div class="social-area">
+                    <div class="social-network">LET’S CONNECT</div>
+                    <div class="social-icon">
+                        <a href="#" class="twitter"><img src="../setup-and-mobile-version-skeleton/images/Twiter.svg" alt=""></a>
+                        <a href="#" class="linkedin"><img src="../setup-and-mobile-version-skeleton/images/Linkedin.svg" alt=""></a>
+                        <a href="#" class="M"><img src="../setup-and-mobile-version-skeleton/images/M.svg" alt=""></a>
+                        <a href="#" class="C"><img src="../setup-and-mobile-version-skeleton/images/C.svg" alt=""></a>
+                        <a href="#" class="2f"><img src="../setup-and-mobile-version-skeleton/images/2f.svg" alt=""></a>
+                    </div>
```

Comment:
- [ ] Kindly move your social icons into <li> tags inside an <ul> tag. It is more semantically accurate and easier to handle.

Reply:
@JuliCarracedo hi ok

Reply:
@JuliCarracedo what you mean, please

Reply:
@JuliCarracedo  hi Sir I have updated your review suggestions in the code please help to review it